lib/OpenQA/BuildResults.pm

```diff
@@ -131,13 +131,24 @@ sub compute_build_results ($group, $limit, $time_limit_days, $tags, $subgroup_fi
         return \%result;
     }
 
+    # build sorting
+    # 1: sort by build name (default)
+    # 0: sort by newest job in build
+    # 2: sort by oldest job in build
+    my $buildver_sort_mode = 1;
+    $buildver_sort_mode = $group->build_version_sort if $group->can('build_version_sort');
+    my $sort_column = 'lasted_job';
+    if ($buildver_sort_mode == 2) {
+        $sort_column = 'oldest_job';
+    }
```

Comment:
This could be more nicely assigned using the ternary operator.

```suggestion
    my $sort_column = $buildver_sort_mode == 2 ? 'oldest_job' : 'lasted_job';
```